pycheribuild/projects/cross/cheribsd.py

```diff
@@ -2244,6 +2256,10 @@ def process(self) -> None:
         release_args.set_env(INSTALL="sh " + str(self.source_dir / "tools/install.sh"))
         release_args.set_env(XZ_CMD=str(self.objdir / "tmp/legacy/usr/bin/xz -T 0"))
 
+        if self.build_vm_images:
+            release_args.set_env(WITH_CLOUDWARE="yes")
+            release_args.set_env(WITHOUT_QEMU="yes")
```

Comment:
Do these need to be in the env, or does set_with_options(FOO=False) work?

Reply:
Switched